start-instances

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
out of curiosity whats this function provide? it doesn't seem to be called anywhere

Reply:
It's a convenient script for starting multiple runners, so it will be run manually.